docs/import-from-whitehall.md

```diff
@@ -0,0 +1,110 @@
+# Importing documents from Whitehall to Content Publisher
+
+There are two ways to import documents from Whitehall:
+- a single document
+- documents of a specific type or subtypes from a single publishing organisation
+
+## Running an import
+
+### Single document import
+
+This will import a single document and all its editions from Whitehall into Content Publisher.
+
+```
+rake import:whitehall_document[1234]
+```
+
+Where `1234` is the Whitehall document ID.  After running the rake task, a URL will be printed to the console which allows the status of import to be examined.
+
+Note that the ID in Whitehall's internal URLs is the edition number (e.g. for `/government/admin/publications/1034451` the edition is `1034451`).  The document ID can be obtained on a Rails console, using:
+
+```
+Edition.find(1034451).document.id
+```
+
+### Import documents of a specific type (and optional subtypes) from a single publishing organisation
```

Comment:
I feel like we should suggest that people pass in the subtypes by default rather than saying it's optional.

This seems to have caught Emma out (see https://trello.com/c/nibnOHkx/1538-dont-export-government-responses-from-whitehall) where I've suggested we might want to do more with defaults.

Reply:
I've removed the words optional.  We can update this if any changes are made to the optional nature of the subtypes.